src/cascadia/TerminalSettingsEditor/Actions.xaml

```diff
@@ -0,0 +1,189 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+the MIT License. See LICENSE in the project root for license information. -->
+<Page
+    x:Class="Microsoft.Terminal.Settings.Editor.Actions"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Terminal.Settings.Editor"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    xmlns:SettingsModel="using:Microsoft.Terminal.Settings.Model"
+    mc:Ignorable="d">
+
+    <Page.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="CommonResources.xaml"/>
+            </ResourceDictionary.MergedDictionaries>
+
+            <local:StringIsEmptyConverter x:Key="CommandKeyChordVisibilityConverter"/>
+
+            <!-- Template for actions. This is _heavily_ copied from the command
+            palette, with modifications:
+                * We don't need to use a HighlightedTextControl, because we're
+                  not filtering this list
+                * We don't need the chevron for nested commands
+                * We're not displaying the icon
+                * We're binding directly to a Command, not a FilteredCommand
+
+                If we wanted to reuse the command palette's list more directly,
+                that's theoretically possible, but then it would need to be
+                lifted out of TerminalApp and either moved into the
+                TerminalSettingsEditor or moved to it's own project consumed by
+                both TSE and TerminalApp.
+             -->
+            <DataTemplate x:Key="GeneralItemTemplate" x:DataType="SettingsModel:Command">
+
+                <!-- This HorizontalContentAlignment="Stretch" is important
+                        to make sure it takes the entire width of the line -->
+                <ListViewItem HorizontalContentAlignment="Stretch"
+                              AutomationProperties.Name="{x:Bind Name, Mode=OneWay}"
+                              AutomationProperties.AcceleratorKey="{x:Bind KeyChordText, Mode=OneWay}">

```

Comment:
This one is technically wrong. It suggests that pressing this chord _right now_ will activate this action, but what we're providing is more informative than actionable.

I think for this one we should _not_ use AcceleratorKey and instead remove the Raw property on the border/textblock that contains the binding.

Reply:
Weird. Removing the Accelerator key, and the Raw from the Border&TextBox means that _nothing_ gets read out at all. Is there a better thing I should bind to?

Reply:
Wait, seriously? Frick, just put it back on the AcceleratorKey then. Just so long as it can be read.